tracer/src/Datadog.Trace/Tags.cs

```diff
@@ -454,5 +454,11 @@ public static class Tags
         internal const string CouchbaseOperationCode = "couchbase.operation.code";
         internal const string CouchbaseOperationBucket = "couchbase.operation.bucket";
         internal const string CouchbaseOperationKey = "couchbase.operation.key";
+
+        internal const string UserEmail = "usr.email";
+        internal const string UserName = "usr.name";
+        internal const string UserId = "usr.id";
+        internal const string UserSessionId = "usr.session_id";
+        internal const string UserRole = "usr.role";
```

Comment:
I wonder if we should start grouping these to keep them organized.

Maybe a nested class?
```csharp
public static class User
{
    Email = "usr.email";
    // ...
}
```

Or maybe start splitting the AppSec, CI, Tracer tags into separate files or something? Just brainstorming here. Doesn't need to be in this PR, either.